### DISCUSSION TITLE
SSL error: cipher operation failed?

### DISCUSSION THREAD

**Discussion by jeffersonkr** [psycopg/psycopg2#1615] [Q&A]
Hi im using psycopg2 = "2.9.5", and sometimes my queries is returning this error:

```
Error (psycopg2.OperationalError) SSL error: cipher operation failed
```
some one can explain what does this error means?

**Comment by dvarrazzo** [comment]
Never seen it. It must be something on your network, especially if it happens occasionally.

**Reply by jeffersonkr** [reply]
hello dvarrazzo first of all thanks for reply, I'm running on EC2 and RDS from AWS, I also checked the latency of the pods and they are stable

**Comment by jeffersonkr** [comment]
bumpep flask version then the error is gone.
